server/models/Transaction.js

```diff
@@ -437,11 +439,22 @@ export default (Sequelize, DataTypes) => {
         fromCollective.currency,
         transaction.createdAt,
       );
+      const oppositeTransactionHostCurrencyFxRate = await getFxRate(
+        transaction.currency,
+        fromCollectiveHost.currency,
+        transaction.createdAt,
+      );
+
       oppositeTransaction = {
         ...oppositeTransaction,
-        HostCollectiveId: fromCollective.HostCollectiveId,
+        HostCollectiveId: fromCollectiveHost.id,
         currency: fromCollective.currency,
+        hostCurrency: fromCollectiveHost.currency,
+        hostCurrencyFxRate: oppositeTransactionHostCurrencyFxRate,
         amount: -Math.round(transaction.netAmountInCollectiveCurrency * oppositeTransactionCurrencyFxRate),
+        amountInHostCurrency: -Math.round(
+          transaction.netAmountInCollectiveCurrency * oppositeTransactionHostCurrencyFxRate,
+        ),
```

Comment:
TODO: what about `*feeInHostCurrency`, what should we do?

Reply:
If we don't convert, they'll be in the original host currency, right?
I think it would be interesting to convert to the oppositeTransaction's host currency, it will be useful later on when we start to collect fees from the host.